README.md

```diff
@@ -1,2 +1,89 @@
 # Faros Events CLI
+
 CLI for reporting events to Faros platform
+
+## `faros_event.sh` Usage Examples
+
+### Sending a Build Event
+
+```sh
+# Using flags
+./faros_event.sh build -k "<api_key>" \
+    --application "<app_name>" \
+    --build_status "<build_status>" \
+    --ci_org "<ci_organization>" \
+    --commit "<commit_sha>" \
+    --repo "<vcs_repo>" \
+    --pipeline "<ci_pipeline>" \
+    --vcs_source "<vcs_source>" \
+    --vcs_org "<vcs_organization>" \
+    --print_event
+
+# Using environment variables
+FAROS_API_KEY="<api_key>" \
```

Comment:
1. I think we should prefix our env vars with `FAROS_` to avoid conflict with any other env vars in the CI/CD systems. 

2. Better explain the convention (which we should have) in the section below. For example:

### Arguments passing

You can also use environment variables to pass arguments into the script by simply capitalizing the argument name and prefixing it with `FAROS_`. E.g. `--commit_sha` becomes `FAROS_COMMIT_SHA`, `--vcs_org` becomes `FAROS_VCS_ORG`.

```
// example command with a mixed set of args
```